app/serializers/crate.js

```diff
@@ -10,4 +12,23 @@ export default class CrateSerializer extends ApplicationSerializer {
 
     return super.extractRelationships(...arguments);
   }
+
+  normalizeQueryResponse(_store, _modelClass, payload) {
+    // We don't want existing relationships overwritten by results with null values.
+    // See: https://github.com/rust-lang/crates.io/issues/10711
+    if (payload.crates) {
+      payload = {
+        ...payload,
+        crates: payload.crates.map(crate => {
+          for (const rel of SKIP_NULL_FIELDS) {
+            if (crate[rel] == null) {
```

Comment:
```suggestion
            if (crate[rel] === null) {
```

I guess we should be explicit here?

Reply:
yeah, fair enough!